.travis.yml

```diff
@@ -15,7 +15,7 @@ addons:
 
 before_install:
   - cd linux/
-  - npm i -g npm@5.2.0
+  - npm i -g npm@latest
```

Comment:
The previous bug affecting the npm v5.3 has been fixed. We can now use the latest version v5.4.1